scopesim/effects/spectral_trace_list.py

```diff
@@ -142,6 +147,34 @@ def make_spectral_traces(self):
 
         self.spectral_traces = spec_traces
 
+    def update_meta(self):
+        """
+        Update fov related meta values
+
+        The values describe the full extent of the spectral trace
+        volume in wavelength and space
+        """
+        wlim, xlim, ylim = [], [], []
+        for thetrace in self.spectral_traces.values():
+            fov = thetrace.fov_grid()
+            if 'wave_min' in fov:
+                wlim.extend([fov['wave_min'], fov['wave_max']])
+            if 'x_min' in fov:
+                xlim.extend([fov['x_min'], fov['x_max']])
+            if 'y_min' in fov:
+                ylim.extend([fov['y_min'], fov['y_max']])
+
+        if wlim != []:
+            self.meta['wave_min'] = np.min(wlim)
+            self.meta['wave_max'] = np.max(wlim)
+        if xlim != []:
+            self.meta['x_min'] = np.min(xlim)
+            self.meta['x_max'] = np.max(xlim)
+        if ylim != []:
```

Comment:
```suggestion
        if wlim:
            self.meta['wave_min'] = np.min(wlim)
            self.meta['wave_max'] = np.max(wlim)
        if xlim:
            self.meta['x_min'] = np.min(xlim)
            self.meta['x_max'] = np.max(xlim)
        if ylim:
```
Empty `list` is `False`, non-empty `list` is `True`, so this expression can be simplified.